src/Web/Sfa.Das.Sas.Web/Controllers/ApprenticeshipController.cs

```diff
@@ -73,7 +73,7 @@ public ActionResult ApprenticeshipOrProvider(bool? retry, bool? isApprenticeship
         [HttpGet]
         public ActionResult SearchResults(ApprenticeshipSearchQuery query)
         {
-            var response = _mediator.Send(query);
+            var response = _mediator.Send(query).Result;
```

Comment:
These calls to `Result` could be a problem using 'old' asp.net?
http://blog.stephencleary.com/2012/07/dont-block-on-async-code.html

Reply:
updated to make sure is only used in async scenarios